resources/elrepo.rb

```diff
@@ -0,0 +1,34 @@
+resource_name :osl_repos_elrepo
+provides :osl_repos_elrepo
+
+### Note: the elrepo repository is only availible on centos 8, using this resource on centos 7 will produce no results
+
+default_action :add
+
+# This property indicates wether the elrepo repo should be enabled
+# If unmanaged the repo file will not be created or changed
+property :elrepo, [true, false], default: true
+
+# This is the default and only action, It will add all availible repos, unless specified in properties above
+action :add do
+  # Install the repo only on Centos 8
```

Comment:
This was only specific to our use case in the base cookbook. In general we may use this on all platforms so this resource should work on both C7 and C8. We'll just keep the logic for it in ``base::packages`` for C8.